plugin/type_uuid/mysql-test/type_uuid/func_uuidv7_plugin.test

```diff
@@ -0,0 +1,30 @@
+--echo #
+--echo # Start of 10.5 tests
+--echo #
+
+--echo #
+--echo # MDEV-32637 Implement native UUID7 function
+--echo #
+
+--vertical_results
+SELECT
+  '----' AS `----`,
+  PLUGIN_NAME,
+  PLUGIN_VERSION,
+  PLUGIN_STATUS,
+  PLUGIN_TYPE,
+  PLUGIN_AUTHOR,
+  PLUGIN_DESCRIPTION,
+  PLUGIN_LICENSE,
+  PLUGIN_MATURITY,
+  PLUGIN_AUTH_VERSION
+FROM INFORMATION_SCHEMA.PLUGINS
+WHERE PLUGIN_TYPE='FUNCTION'
+  AND PLUGIN_NAME IN
+    ('uuidv7')
+ORDER BY PLUGIN_NAME;
+--horizontal_results
+
```

Comment:
There should be some tests that actually call `UUIDv7`

plugin/type_uuid/mysql-test/type_uuid/order.test.

While a straight SELECT On the result will result in unpredicaable output something like inserting:
```
CREATE TABLE t1 (a int primary key not null, u UUID DEFAULT UUIDv7(), unique key(u));
insert into t1(a) select seq from seq_1_to_10;
select a from t1 order by u asc;
```
Ensuring uniqueness and the order of generation is ascending.

Could also pull the version/variant from the generated values and ensure they are correct.

Reply:
Also tests involving replication like mysql-test/suite/rpl/include/rpl_row_UUID.test.